mswindows/osgeo4w/python-grass.bat.tmpl

```diff
@@ -0,0 +1,18 @@
+@echo off
+rem #########################################################################
+rem #
+rem # GRASS initialization bat script for Python environment (OSGeo4W)
+rem #
+rem #########################################################################
+
+rem
+rem Set environmental variables
+rem
+call "%~dp0\o4w_env.bat"
+call "%OSGEO4W_ROOT%\apps\grass\grass@POSTFIX@\etc\env.bat"

```

Comment:
related to my comment above, in _env.bat_

`call %OSGEO4W_ROOT%\bin\py3_env.bat`

ok, we have to be careful not to invoke things twice.

maybe some system refactoring needed.